### ISSUE TITLE
Doesn't work on 0.19

### ISSUE THREAD

**Issue by vodik** [gulp-elm/gulp-elm#35]
`elm-make` is gone, replaced with just a single `elm` binary.

As a shim, installing this shell script as `elm-make` technically makes it works:

```bash
#!/bin/bash
shift  # swallow --yes, its gone
[[ "$@" ]] && exec node_modules/.bin/elm make "$@"
```

**Comment by bdukes** [comment]
Thanks for documenting this workaround @vodik

I took a first stab at getting _gulp-elm_ to work for Elm 0.19, but wasn't sure of a good way to do that without breaking support for older versions of Elm.  

My current train of thought is to check for the existence of `elm.json` or `elm-package.json` and adjust the default settings and command line arguments based on that.  But it's a fairly busy season for me, not sure when I'll get that done

**Comment by vodik** [comment]
Another approach could be to look for `elm` and decide if its either local or global, and then look for `elm-make` to be next to it. It should be safe to assume that no `elm-make` means 0.19.

I can contribute patches if they're welcome then.

**Comment by vodik** [comment]
Or I guess another option is to break backwards comparability, just like Elm did, and let the current release be the last one that runs with 0.18? It seems like its rather mature. I don't know how palatable that is to you though.

**Comment by ggmartinsen** [comment]
Blocked on this as our last task to get CI up and running for 0.19.
Any ETA?

**Comment by bdukes** [comment]
@ggmartinsen I've published 0.8.0 of _gulp-elm_ which supports (only) Elm 0.19.

Let us know if you see any issues with it.
